README.md

```diff
@@ -0,0 +1,48 @@
+# Hello Microverse Project
+
+> Description the project.
```

Comment:
- [ ] Kindly write a proper description for your project so that your readme can be more professional 👍🏽

Reply:
Thank you for the feedback. I've made the change.